var/spack/repos/builtin/packages/mfem/package.py

```diff
@@ -45,10 +45,14 @@ class Mfem(Package):
     # other version.
     version('develop', branch='master')
```

Comment:
```suggestion
    version('master', branch='master')
```

Reply:
Changing `develop` to `master` will require changes in other packages (like `laghos`, `remhos`) and offline scripts that people may already have, so I'd prefer to keep it as is.

@tzanio, @acfisher, what do you think?

Reply:
I'd prefer to keep `develop` as is too (it is also semantically correct).

We can add another version `master` if required.